lib/datadog/core/configuration/settings.rb

```diff
@@ -122,9 +122,16 @@ def initialize(*_)
           # @default `DD_TRACE_DEBUG` environment variable, otherwise `false`
           # @return [Boolean]
           option :debug do |o|
-            o.env Datadog::Core::Configuration::Ext::Diagnostics::ENV_DEBUG_ENABLED
+            o.env [Datadog::Core::Configuration::Ext::Diagnostics::ENV_DEBUG_ENABLED,
+                   Datadog::Core::Configuration::Ext::Diagnostics::ENV_OTEL_LOG_LEVEL]
             o.default false
             o.type :bool
+            o.env_parser do |value|
+              if value
+                value = value.strip.downcase
+                ['true', '1', 'debug'].include?(value)
```

Comment:
Can you document which of these values are for otel? For example, in this case, I think `debug` is for otel.